Jenkinsfile_CNP

```diff
@@ -1,28 +1,25 @@
 #!groovy
 
-properties([
-
-  parameters([
-    string(name: 'URL_TO_TEST', defaultValue: 'http://am-lib-test-aat.service.core-compute-aat.internal', description: 'The URL you want to run these tests against')
-  ])
-])
 
 @Library("Infrastructure")
 
-import uk.gov.hmcts.contino.AppPipelineConfig
 
 def type = "java"
 def product = "am"
 def component = "lib-test"
 
 def secretId = "am-lib-test-${params.ENVIRONMENT}"
```

Comment:
this won't work for the `master` pipeline because there are 2 environments (`aat` and `prod`) and this will hard code the vault name to one thing for the entire pipeline